Index: src/bai2/entity/QLGV_Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/bai2/entity/QLGV_Main.java b/src/bai2/QLGV_Main.java
rename from src/bai2/entity/QLGV_Main.java
rename to src/bai2/QLGV_Main.java
--- a/src/bai2/entity/QLGV_Main.java	(revision 1aeffb566d3324087cb460dc07da750be08d6573)
+++ b/src/bai2/QLGV_Main.java	(date 1659747352111)
@@ -1,11 +1,11 @@
-package bai2.entity;
+package bai2;
 
-import QlyGiaoVien.SalaryMangament;
+import bai2.entity.Teacher;
+import bai2.service.SalaryMangament;
 import QlyGiaoVien.Subject;
 import QlyGiaoVien.Teacher;
 import Tool.TableRow;
 
-import javax.security.auth.Subject;
 import java.util.Arrays;
 import java.util.InputMismatchException;
 import java.util.Scanner;
Index: src/bai2/service/test1.java
===================================================================
diff --git a/src/bai2/service/test1.java b/src/bai2/service/test1.java
deleted file mode 100644
--- a/src/bai2/service/test1.java	(revision 1aeffb566d3324087cb460dc07da750be08d6573)
+++ /dev/null	(revision 1aeffb566d3324087cb460dc07da750be08d6573)
@@ -1,4 +0,0 @@
-package bai2.service;
-
-public class test1 {
-}
Index: src/bai2/entity/SalaryMangament.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/bai2/entity/SalaryMangament.java b/src/bai2/service/SalaryMangament.java
rename from src/bai2/entity/SalaryMangament.java
rename to src/bai2/service/SalaryMangament.java
--- a/src/bai2/entity/SalaryMangament.java	(revision 1aeffb566d3324087cb460dc07da750be08d6573)
+++ b/src/bai2/service/SalaryMangament.java	(date 1659747352142)
@@ -1,4 +1,4 @@
-package QlyGiaoVien;
+package bai2.service;
 
 public class SalaryMangament {
 
